terminal.go

```diff
@@ -873,6 +874,7 @@ func (g *LightningTerminal) setUpLNDClients(lndQuit chan struct{}) error {
 				BlockUntilUnlocked:    true,
 				CallerCtx:             ctxc,
 				CheckVersion:          minimalCompatibleVersion,
+				RPCTimeout:            g.cfg.LndRPCTimeout,
```

Comment:
Hmmmmm I want to also note that this modifies the timeout used in `lndclient` during the initial chain sync. If 

We've had many users which have experienced timeout issues during the initial chain sync, and from my experience i'd sometimes get a response after like 20 seconds when the syncing issues occur.
Here's an issue report for that specific issue:
https://github.com/lightningnetwork/lnd/issues/9108

I do believe that setting this value to a too low value would likely makes those syncing issues even worse when they actually occur.

Therefore: would we like to set a lower limit for this value, just so that some users don't set it to like 1 second? Or I'd at least mention in the config flag docs that it's recommended to not set this value to a too low value.

Reply:
Yeah, good point. I added a startup check and also mentioned it in the release notes.

Reply:
Awesome thanks :pray:!